src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.spec.ts

```diff
@@ -742,7 +731,7 @@ describe('PoolFormComponent', () => {
   describe('submit - create', () => {
     const setMultipleValues = (settings: {}) => {
```

Comment:
The FormHelper has an implementation of setMultipleValues which isn't used here, but it looks as if that was intended.

Reply:
But it could also already be used in this spec file!

Reply:
Yes it could, haven't seen that as I split the PR up after the erasure code profile was done. While developing this additional method was created.

I will amend it.